x-pack/plugins/security_solution/public/management/cypress/e2e/policy/policy_experimental_features_disabled.cy.ts

```diff
@@ -12,7 +12,8 @@ import type { IndexedFleetEndpointPolicyResponse } from '../../../../../common/e
 import { login } from '../../tasks/login';
 import { createAgentPolicyTask, getEndpointIntegrationVersion } from '../../tasks/fleet';
 
-describe('Disabled experimental features on: ', { tags: '@ess' }, () => {
+// We need a way to disable experimental features in the Cypress tests
+describe.skip('Disabled experimental features on: ', { tags: '@ess' }, () => {
```

Comment:
Should we remove those instead?

Reply:
Its an open question. What's your take? If we believe that it might so happen that we will turn it off in the future then I believe we should test scenarios when its disabled. I would tackle the way to do it in another ticket though.